frontend/src/containers/PostsView.tsx

```diff
@@ -96,6 +97,7 @@ class Posts extends React.Component<Props> {
                             <p>{p.content}</p>
                             <PostComments postId={p.id} />
                             {/* TODO here the same as for CreatePostForm */}
+                            //@ts-ignore
```

Comment:
Have you checked the type declaration for Form.create? Maybe it allows to pass additional interface from outside.

Reply:
I found a way how to fix it a few days ago - I'll include it in the upcoming UX improvement PR.